generic-qemu.toml

```diff
@@ -0,0 +1,4 @@
+# vyos build for gerneric qemu/kvm environments like proxmox. 
```

Comment:
Typo: "gerneric".
"VyOS" should be properly capitalized, too, I suppose. :)

Reply:
I will update it with the next commit